src/patternfly/components/EmptyState/empty-state.scss

```diff
@@ -71,17 +61,26 @@
   color: var(--pf-c-empty-state__body--Color);
 }
 
+@at-root .pf-c-empty-state__content > .pf-c-button.pf-m-primary,
+.pf-c-empty-state__primary {
+  margin-top: var(--pf-c-empty-state__primary--MarginTop);
+
+  + .pf-c-empty-state__secondary {
+    margin-top: var(--pf-c-empty-state__primary--secondary--MarginTop);
+  }
+}
+
 .pf-c-empty-state__secondary {
   display: flex;
   flex-wrap: wrap;
   justify-content: center;
   margin-top: var(--pf-c-empty-state__secondary--MarginTop);
-  margin-right: var(--pf-c-empty-state__secondary--MarginRight);
   margin-bottom: var(--pf-c-empty-state__secondary--MarginBottom);
 
   > .pf-c-button {
-    margin-right: var(--pf-c-empty-state__secondary--c-button--MarginRight);
-    margin-bottom: var(--pf-c-empty-state__secondary--c-button--MarginBottom);
+    margin-right: var(--pf-c-empty-state__secondary__c-button--MarginRight);
```

Comment:
I wonder if we should also support a plain `<a>` link that doesn't use `pf-c-button` ?